server/models.ts

```diff
@@ -7,8 +7,10 @@ if (process.env.NODE_ENV !== 'production' && process.env.NODE_ENV !== 'test') {
 	require(path.join(process.cwd(), 'config.js'));
 }
 
-const useSSL = process.env.DATABASE_URL!.indexOf('localhost') === -1;
-export const sequelize = new (Sequelize as any)(process.env.DATABASE_URL, {
+// @ts-expect-error
```

Comment:
These `@ts-expect-error` comments are related to some goofy stuff where the test runner can't parse this file as TypeScript. It's actually a reversion of a change I made in an earlier PR [here](https://github.com/pubpub/pubpub/pull/1462/files#diff-8e1cca28a50d0636bee119e2cb5f80e920aa0f85f584e285a939a8cdb99f5415R10) and I think the whole thing will just come out in the wash without any changes once we merge them all together.